source/blog/2020-10-27-this-month-in-solidus-september-2020.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+title: "This Month in Solidus: September 2020"
+date: 2020-10-27
+tags: Solidus
+author: Sean Denny
+cover_image: /blog/shared/images/solidus-monthly-update.png
+description: >
+  Dive in and see everything that happened in the Solidus world in September 2020: Bug fixes, convenient changes, and information on our next Solidus Meetup. Also: Solidus 2.11 is live!
+---
+
+Hello everyone, and welcome to This Month in Solidus — September 2020! Let's dive in to what's new with the platform.
+
+### Extension updates for September
+
+- **NextJS Storefront:** There's a new Solidus storefront on the block - [`solidus_nextjs_frontend`](https://github.com/nebulab/solidus_nextjs_frontend)! We're using GraphQL and NextJS to bring you the snappiest frontend we can build. There's still some work to do, but keep an eye out on this project. We've completed the basic structure and GraphQL SDK, and we're working on the checkout now. Thanks to Andrea Vassallo, Rainer Dema, Dino Hamzić, and Alessio Rocco from Nebulab.
+- **Data Importer**: Version 0.2.0 is officially out! With this release of [`solidus_importer`](https://github.com/solidusio-contrib/solidus_importer), the order importer has been fully hooked in. Now you'll be able to import orders from anywhere - using the [Shopify CSV](https://help.shopify.com/en/manual/products/import-export/using-csv) format. Thanks to Mattia Roccoberton, Flavio Auciello, and Elia Schito from Nebulab!
+- **Data Feeds**: With the release of the data importer, we decided that we also needed a way to export data. This is what the [`solidus_feeds`](https://github.com/nebulab/solidus_feeds/) extension handles! It's still a work in progress, but it adds support for generating Google Merchant feeds, and publishing them to S3. We're releasing our first version soon, then we'll be adding more feeds and uploaders. Stay tuned for more info - and thanks to Elia Schito from Nebulab.
```

Comment:
Can you also mention Nicolò Rebughini from Nebulab here? He did a significant amount of work. 🙂